src/studio/src/designer/backend/Controllers/TextsController.cs

```diff
@@ -30,12 +32,40 @@ public class TextsController : ControllerBase
         /// <summary>
         /// Initializes a new instance of the <see cref="TextsController"/> class.
         /// </summary>
-        /// <param name="textsService">The languages service.</param>
+        /// <param name="textsService">The texts service.</param>
         public TextsController(ITextsService textsService)
         {
             _textsService = textsService;
         }
 
+        /// <summary>
+        /// Endpoint for editing text file for a specific language.
+        /// </summary>
+        /// <param name="org">Unique identifier of the organisation responsible for the app.</param>
+        /// <param name="repo">Application identifier which is unique within an organisation.</param>
+        /// <param name="languageCode">Language identifier specifying the text file to read.</param>
+        /// <param name="jsonText">New content from request body to be added to the text file.</param>
+        /// <remarks>If duplicates of keys are tried added the
+        /// deserialization to dictionary will overwrite the first
+        /// key:value pair with the last key:value pair</remarks>
+        [HttpPut]
+        [Produces("application/json")]
+        [ProducesResponseType(StatusCodes.Status200OK)]
+        [ProducesResponseType(StatusCodes.Status400BadRequest)]
+        [Route("{languageCode}")]
+        public async Task<ActionResult> EditText(string org, string repo, string languageCode, [FromBody] Dictionary<string, string> jsonText)
+        {
+            string developer = AuthenticationHelper.GetDeveloperUserName(HttpContext);
+
+            await _textsService.EditText(org, repo, developer, languageCode, jsonText);
+
+            return new JsonResult(new
```

Comment:
This has a couple of issues:
1. The http status code should be used by the frontend to determine if the operation was a success or not ie. success=true isn't necessary.
2. Put should return either a 200 Ok with the resource in the body, or a 204 No Content if we don't send the content back.
3. Given the whole response with Success and Message is gone. If we decide we need those then the text 'Språk lagret' is hardcoded to norwegian. While we currently don't support multiple languages in Designer we should discuss this. Either way the UI texts should be handled by the frontend, and at most the backend should return a code for what text to use. But I think the standard status codes should be used as much as possible.